app/routers/event.py

```diff
@@ -37,13 +38,39 @@
 )
 
 
-@router.get("/edit")
+class EventModel(BaseModel):
+    title: str
+    start: dt
+    end: dt
+    content: str
+    owner_id: int
+    location: str
+
+
+@router.get("/events")
+async def get_events(session=Depends(get_db)):
+    return session.query(Event).all()
+
+
+@router.post("/create_event")
+async def create_event_api(event: EventModel, session=Depends(get_db)):
+    create_event(db=session,
+                 title=event.title,
+                 start=event.start,
+                 end=event.start,
+                 content=event.content,
+                 owner_id=event.owner_id,
+                 location=event.location)
+    return f'Successfully created {event.title} event'
```

Comment:
Prefer returning a json: `{'success': True}` (if you want it to be perfect, return the details of the created event)